qcodes/instrument_drivers/Keysight/N51x1.py

```diff
@@ -13,6 +13,27 @@ class N51x1(VisaInstrument):
     def __init__(self, name: str, address: str, min_power: int = -144, max_power: int = 19, **kwargs: Any):
         super().__init__(name, address, terminator='\n', **kwargs)
 
+        self._options = self.ask("*OPT?")
```

Comment:
the comment in the old code makes me think that the instrument response is a comma-separated list of numbers, right? if so, then it would've been better to split the returned string into an actualy list of strings with numbers. the ``if f_option in self._options`` check would still work, but would be more robust, so that e.g. option `50101` does not get mistakenly treated as option `501` (but of course likely option `50101` does not exist, so this all is likely not a problem in real life)
```python
self._options = self.ask("*OPT?").split(",")
```